docs/tutorial.md

```diff
@@ -668,6 +671,9 @@ using the CSV file works as you would expect:
 # adds mutations to the seq object
 >>> ma.add_mutations(seq)
 
+# adds mutations and the corresponding metadata to the seq obejct
```

Comment:
yep - but can we have this example as a comment rather than an actually run line?

I'm asking this because the tutorial is constructed so that one can mindlessly paste the lines from it and run it. If we leave two of these lines we will annotate the mutations twice

Reply:
Yes, in relation to this, is it alright if I remove >>> from all the code pieces? It would make it much easier to copy.